### PR TITLE
CRM 13823 admin status page

### PR DESCRIPTION
**Pull request by jdax**
Admin status page will
display all current warnings, color-coded by severity
allow users to snooze or hush those message
display a footer that shows current system settings with link to the status page

https://issues.civicrm.org/jira/browse/CRM-13823

### CODE REVIEW

**Comment by kurund** [comment]
jenkins, test this please

**Comment by kurund** [comment]
@jdax looks like there are few more style errors: https://test.civicrm.org/job/CiviCRM-Core-PR/5954/checkstyleResult/new/

**Comment by jdax** [comment]
I believe I've fixed them for real, this time.

**Comment by agh1** [comment]
Sorry @jdax I steered you wrong.  It should be 'id' not $id at https://github.com/jdax/civicrm-core/blob/CRM_13823_Admin_Status_Page/api/v3/System.php#L116

File: CRM/Utils/Check.php [hunk 1]
```diff
@@ -91,7 +91,7 @@ public function showPeriodicAlerts($messages = NULL, $filter = array(__CLASS__,
         if (count($statusMessages)) {
           if (count($statusMessages) > 1) {
             $statusTitle = ts('Multiple Alerts');
-            $statusMessage = '<ul><li>' . implode('</li><li>', $statusMessages) . '</li></ul>';
+            $statusMessage = 'Please check your <a href="../civicrm/a/#/status">status page</a> for a full list and further details. <ul><li>' . implode('</li><li>', $statusMessages) . '</li></ul>';
```
Comment (kurund) [inline CRM/Utils/Check.php]: ts( ) is missing here

File: ang/crmStatusPage/StatusPageCtrl.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  angular.module('statuspage').controller('statuspageStatusPage',
+    function($scope, $location, crmApi, crmStatus, crmUiHelp, statuses, crmNavigator, preferences) {
+    // The ts() and hs() functions help load strings for this module.
+    var ts = $scope.ts = CRM.ts('statuspage');
+    var hs = $scope.hs = crmUiHelp({file: 'CRM/statuspage/StatusPage'}); // See: templates/CRM/statuspage/StatusPage.hlp
+
+    $scope.path = $location.path();
+    $scope.navigator = crmNavigator;
+    $scope.statuses = statuses;
+    console.log(preferences);
```
Comment (kurund) [inline ang/crmStatusPage/StatusPageCtrl.js]: remove the debug

**Pull request merged**